client_cards_plugin.php

```diff
@@ -18,6 +18,8 @@ public function __construct()
         Language::loadLang('client_cards_plugin', null, dirname(__FILE__) . DS . 'language' . DS);
 
         $this->loadConfig(dirname(__FILE__) . DS . 'config.json');
+
+        $this->client_uri = WEBDIR . Configure::get('Route.client') . '/';
```

Comment:
You test this?  Adding WEBDIR here breaks it for me.  Pretty sure that part is already taken care of by the public_uri appended in the core

Reply:
Yes, I tested it and worked for me. But yes, you're right, public_uri takes care of the web dir. Updated.

Reply:
Got it, it's probably because my installation is in a sub directory